### ISSUE TITLE
[Quest] The Desert Fox - Uldum

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Blackwing-Lair#4117]
**Links:**

quest https://www.wowhead.com/quest=27939/the-desert-fox
https://www.wowhead.com/item=62777/the-desert-fox

![desert fox 1](https://user-images.githubusercontent.com/39439201/71310084-3f66cd80-2410-11ea-9668-399f5350f97e.jpg)

**What is happening:**
- Fox i found wasnt clickable so cant complete 

**What should happen:**
https://www.wowhead.com/quest=27939/the-desert-fox#videos:id=88489

**Comment by Druhl** [comment]
Is it possible open-issue broken quests like this one might be "flagged as bugged" in-game so they could respond to the .bq command? Especially like this one where it breaks or prevents a chain.

Thanks!

**Comment by Echinaceax** [comment]
@Druhl best to ask this in discord gitH channel :)

**Comment by Butterjib** [comment]
Quest is bugged,  fox not interactable.  .bq command not working #27939

Thanks

**Comment by Echinaceax** [comment]
@Butterjib 
yes this quest isnt on the list for the command, we only added 176 quests to the .bq comment to help players a little

**Comment by golosomie** [comment]
pending for .bq command
